README.md

```diff
@@ -2,8 +2,11 @@
 
 Minimal bootnode utility, with ENR options, geth discv5 logging, and persisted local node DB. 
 
+And a minimal HTTP endpoint, to serve the local ENR, in common base64 ENR representation, on the `/enr` route.
+
 Options:
 ```
+      --api-addr string        Address to bind HTTP API server to. API is disabled if empty. (default "0.0.0.0:80")
```

Comment:
default is `8000` looks like - further in the code

Reply:
Oh whoops, forgot to update that. Wasn't sure about the best default

Reply:
Fixed in 01f4bdc3ae898cfaeff00c161dd010f38e3a48a1